plugins/org.wso2.developerstudio.eclipse.gmf.esb.diagram/src/org/wso2/developerstudio/eclipse/gmf/esb/diagram/custom/deserializer/PropertyMediatorDeserializer.java

```diff
@@ -104,6 +104,7 @@ public PropertyMediator createNode(IGraphicalEditPart part, AbstractMediator med
             if (propertyMediator.getExpression() != null) {
                 SynapsePath xpath = propertyMediator.getExpression();
                 NamespacedProperty namespaceProp = createNamespacedProperty(xpath);
+                namespaceProp.setSupportJsonPaths(true);
```

Comment:
Are we supporting JSON paths for the property mediator?

Reply:
Yes. We are supporting.